pages/treeherder.py

```diff
@@ -61,6 +61,7 @@ class ResultSet(PageRegion):
 
         _datestamp_locator = (By.CSS_SELECTOR, '.result-set-title-left > span a')
         _jobs_locator = (By.CLASS_NAME, 'job-btn')
+        _pin_all_jobs_locator = (By.CSS_SELECTOR, '.result-set .glyphicon-pushpin')
```

Comment:
Sorry, error on my part... if it's just a class name you don't want a CSS selector:

```python
_pin_all_jobs_locator = (By.CLASS_NAME, 'glyphicon-pushpin')
```